src/todolist/TaskServiceImpl.java

```diff
@@ -1,43 +1,47 @@
 package todolist;
 
-import java.util.Locale;
+import java.util.Collection;
+import java.util.stream.Collectors;
 
 public class TaskServiceImpl implements TaskService {
-    private TaskRepository repository;
-
-    public TaskServiceImpl(TaskRepository repository) {
-        this.repository = repository;
-    }
-
-    @Override
-    public void addTask(String[] parameters) {
-        String title = String.join(" ", parameters);
-        String status = "open";
-        Task task = new Task(title, status);
-        repository.addTask(task);
-    }
-
-    @Override
-    public void listTask(Status status) {
-        String statusList = status.toString().toLowerCase(Locale.ROOT);
-        repository.listTask(statusList);
+  
+  private TaskRepository repository;
+  
+  public TaskServiceImpl(TaskRepository repository) {
+    this.repository = repository;
+  }
+  
+  @Override
+  public void addTask(String[] parameters) {
+    String title = String.join(" ", parameters);
```

Comment:
Этого не было в требованиях, но я бы проверял, что 
```suggestion
    String title = String.join(" ", parameters);
    if ( !title.isBlank() ) {
        throw new IllegalArgumentException("Title should not be null");
    }
```